### DISCUSSION TITLE
Can Fabric loader be installed on the Minecraft Launcher found in the Microsoft Store?

### DISCUSSION THREAD

**Discussion by haykam821** [FabricMC/fabric-api#1814] [Player support] [answered]
There exists [a version of the Minecraft Launcher](https://www.xbox.com/en-us/games/store/Minecraft-Launcher/9PGW18NPBZV5) in the Microsoft Store, which is used to launch Minecraft: Java Edition with Xbox Game Pass for PC. Is Fabric loader installable for the version of Minecraft: Java Edition accessible with it?

**Comment by warjort** [comment]
I haven't tried it myself, but the FAQ suggests it should still work?

It sounds like you need to manually copy things from the old install to the new install due to changed locations and config file names.
**Make backups before you change anything!**

From https://help.minecraft.net/hc/en-us/articles/4412261881229-Minecraft-Launcher-for-Windows-

````
Will my Minecraft: Java Edition profiles/installations transfer to the new Launcher?

If your profiles/installations have not copied over, please follow these steps:

Open the %appdata%/minecraft folder
Create a copy of launcher_profiles.json
Delete launcher_profiles_microsoft_store.json
Rename the copied version of launcher_profiles.json to launcher_profiles_microsoft_store.json
````
and
````
Will my mods carry over to the new launcher? 

If you use launcher or game mods, you will need to migrate them to the new Minecraft Launcher for Windows’ installation location 
before you uninstall your old launcher. Please check with your mod developer for more detail. 
````

If you hit problems, I would suggest you raise any issues here (the actual fabric loader project):
https://github.com/FabricMC/fabric-loader

**Comment by modmuss50** [comment, marked as answer]
Yes, you must use installer `0.9.0` or newer.
